app/views/hyrax/base/_paginated_works.html.erb

```diff
@@ -0,0 +1,31 @@
+<table id="show-page-table" class="display" style="width:50%">
```

Comment:
Add `role="presentation"` to `<table>` to hint to screen readers this is a layout table